Resources/Audio/Voice/Talk/license.txt

```diff
@@ -23,3 +23,7 @@ buwoo_exclaim (Renamed to slime_exclaim.ogg)
 all taken from
 https://github.com/goonstation/goonstation/tree/eb3e7df6292d23f6af2f18b4372d3a8ba4b0fda7/sound/misc/talk
 licensed under CC BY-NC-SA 3.0
+arachnid.ogg
+arachnid_ask.ogg
+arachnid_exclaim.ogg
+arachnid voices by PixelTK licensed under CC-BY-4.0
```

Comment:
Needs to be in attributions.yml

Reply:
There doesn't seem to be a attributions.yml in Resources\Audio\Voice\Talk, should I just make one and move all the stuff I did there?

Reply:
Yeah

Reply:
right did that.